### ISSUE TITLE
There is no LICENSE on this repository

### ISSUE THREAD

**Issue by akhileshthite** [dogecoin/dogecoin.com#200]
I don't know the reason why no LICENSE, but it's good to have one. There is a MIT LICENSE on the original dogecoin repository.

**Comment by NeilShah2026** [comment]
@AkhileshThite Websites (most website anyways) do not need a License. Something like Dogecoin on the other hand does.

**Comment by langerhans** [comment]
Not a lawyer so not actually sure how this would work. Especially re-licensing existing code comes with some problems depending on the license that's picked. Would love for others to chime in who have more experience than I do.

**Comment by NeilShah2026** [comment]
@langerhans 
You do not need a lawyer to get an MIT License. An MIT license, for open-source projects like Dogecoin, would give users the ability to reuse the code for any purpose.

I found this that sums it up better than I can do:
An MIT license gives users express permission to reuse code for any purpose, sometimes even if code is part of proprietary software.

**Comment by langerhans** [comment]
Yeah I understand the MIT license. The problem is that usually when you re/license existing code you'd need to ask existing contributors for their permission. And I'm not sure if that applies to our case and/or the MIT license.

**Comment by NeilShah2026** [comment]
@langerhans No, not for this website

**Comment by NeilShah2026** [comment]
> Yeah I understand the MIT license. The problem is that usually when you re/license existing code you'd need to ask existing contributors for their permission. And I'm not sure if that applies to our case and/or the MIT license.

(if it was applicable) you would just have a contributors guide: #196
